src/AutoMapper/QueryableExtensions/ProjectionExpression.cs

```diff
@@ -87,7 +87,8 @@ public IQueryable<TResult> To<TResult>(IObjectDictionary parameters, params Expr
 
         internal IQueryable<TResult> To<TResult>(IObjectDictionary parameters, MemberPaths memberPathsToExpand)
         {
-            var membersToExpand = memberPathsToExpand.SelectMany(m => m).Distinct().ToArray();
+            IEnumerable<MemberInfo> selectMany = memberPathsToExpand.SelectMany(m => (m ?? new List<MemberInfo>()) as IList<MemberInfo> ?? m /*?.ToList() */);
+            MemberInfo[] membersToExpand = selectMany.Distinct().ToArray();
```

Comment:
Agree. Actually this was my getting-throw hacking piece that I forgot to remove after debug. My fault, I'll reverse this hunk in next update to this PR.